cleanlab_studio/studio/studio.py

```diff
@@ -69,17 +70,17 @@ def _download_cleanlab_columns(
     ) -> pd.DataFrame:
         rows = api.download_cleanlab_columns(self._api_key, cleanset_id, all=True)
         id_col = api.get_id_column(self._api_key, cleanset_id)
-        # TODO actually get _all_ the columns incl e.g., cleanlab_top_labels
-        # and have the API give the column headers rather than this library hard-coding it
```

Comment:
"and have the API give the column headers rather than this library hard-coding it" - can we do that? The backend should just supply the cleanlab columns to the CLI, without duplicating the list of headers here, so when we add new columns, we don't need to change it in two places + release a new version of the CLI.

Reply:
sounds good, this is now resolved